raiden/tests/benchmark/speed_decoding.py

```diff
@@ -1,283 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Well, I added the `locked_amount` field to all messages, the benchmark code would have to be updated to include this new field, while doing the update I realized that most of the messages were out-of-sync anyways, so instead of spending the time fixing the dead code I decided to remove it.

Reply:
It's a good choice. Thank you for doing that.